dev-python/fastimport/fastimport-0.9.12.ebuild

```diff
@@ -13,7 +13,6 @@ SRC_URI="mirror://pypi/${PN::1}/${PN}/${P}.tar.gz"
 
 LICENSE="GPL-2+"
 SLOT="0"
-KEYWORDS=""
```

Comment:
Did you actually test this version?

Reply:
Yeah, tests passed on my end successfully

Reply:
But you do realize that this version has no keywords?

Reply:
Yup, I was just about to add 3.10 to 0.9.11 as well